src/buildHTML.js

```diff
@@ -654,10 +654,60 @@ export default function buildHTML(tree, options) {
     // normal place) so we use an absolute value for vertical-align instead
     bottomStrut.style.verticalAlign = -body.depth + "em";
 
-    // Wrap the struts and body together
-    const htmlNode = makeSpan(["katex-html"], [topStrut, bottomStrut, body]);
+    body.children.unshift(topStrut, bottomStrut);
+
+    return body;
+}
+
+/**
+ * Take an entire parse tree, and build it into an appropriate set of HTML
+ * nodes.
+ */
+export default function buildHTML(tree, options) {
+    // buildExpression is destructive, so we need to make a clone
+    // of the incoming tree so that it isn't accidentally changed
+    tree = JSON.parse(JSON.stringify(tree));
```

Comment:
I didn't realize this.  I wonder how we're mutating `tree`.  🤔

Reply:
I'm curious too! I haven't seen any, but this comment has been in the code for a long time.